website/common/script/ops/buy/buyArmoire.js

```diff
@@ -89,6 +102,8 @@ export class BuyArmoireOperation extends AbstractGoldItemOperation {
 
     removeItemByPath(user, `gear.flat.${drop.key}`);
 
+    this._trackDropAnalytics(user._id, drop.key);
```

Comment:
looks like some tests are failing, maybe this should be inside an `if (this.analytics) {`? (Haven't tested it)

Reply:
Oh, whoops, I fixed this in one place but not the other!